plugins/bnr-dp/fu-bnr-dp-device.c

```diff
@@ -0,0 +1,750 @@
+/*
+ * Copyright 2024-2025 B&R Industrial Automation GmbH
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include <gio/gio.h>
+#include <glib/gtypes.h>
+#include <stddef.h>
```

Comment:
Do you really need all these?  I would think this should be sufficient.

```suggestion
#include <fwupdplugin.h>
```